comps/vectorstores/langchain/vdms/docker-compose-vdms.yml

```diff
@@ -0,0 +1,10 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+version: "3"
+services:
+  vdms-vector-db:
+    image: intellabs/vdms:latest
+    container_name: vdms-vector-db
+    ports:
+      - "55555:55555"
```

Comment:
recommend to use a 4-bit ports number. Some hardware machines may experienced some unexpected issues when having a 5-bits ports

Reply:
changed port to 4-bit number